app/src/main/java/net/gsantner/opoc/model/GsSharedPreferencesPropertyBackend.java

```diff
@@ -71,17 +71,14 @@ public class GsSharedPreferencesPropertyBackend implements GsPropertyBackend<Str
     protected String _prefAppName;
     protected Context _context;
 
-    public GsSharedPreferencesPropertyBackend() {
+    public GsSharedPreferencesPropertyBackend(final Context context) {
```

Comment:
I understand that you prefer AppSettings to instantiate with Context directly. But this here can be left with non-Context requiring constructor, doesn't it? You can also call `super.init(context, name)` in AppSettings constructor.

This here so I can decide at any time to drop this way of using AppSettings and not supply context right away at instantiation. But still keep it as a single instance put somewhere. And...then initalize it there with the context later.

Reply:
Got it. I will rework this back, hopefully keeping the fix intact

Reply:
OK thanks, hope you are okay with that. But times can change fast as you know. Wouldn't be suprised if we sooner or later with Android update we cannot trust the context being available at this stage anymore. Such big breaking things unfortunately happen, just like I never thought 5 years ago that Android file system would get crippled in such a way that it is now.

Reply:
> hopefully keeping the fix intact

generally said, it shouldn't make any difference at all in this case, for execution it doesn't matter if it's codeline, return, call init func, codeline, codeline,codeline...... or codeline,codeline,codeline,codeline,return